src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,32 @@
+import { Portal } from "react-portal";
+import PropTypes from "prop-types";
+
+const Dialog = ({title, onClose, children}) => {
+  return (
+    <div className="dialog">
+        <Portal>
+          <div className="dialog-children">
+            <b className="dialog-close-icon" onClick={onClose}>
+              X
+            </b><br />
+            {title}
+            {children}
+          </div>
+        </Portal>
+    </div>
+  );
+};
+
+Dialog.propTypes = {
+  title: PropTypes.string,
```

Comment:
You can chain any of the above with **isRequired** to make sure a warning is shown if the prop isn't provided.
For example:  `title: PropTypes.string.isRequired`
https://www.npmjs.com/package/prop-types